### PR TITLE
Added test for spawner argument --load-only

### PR DESCRIPTION
**Pull request by Nils-ChristianIseke**
Regarding Issue  #957. As far as i  can see the test for the type_controller argument of the spawner is already implemented. I added a test for testing the --load-only argument for the spawner. I am not sure if it is complete, Feedback is appreciated. I am plannung to add test for all the other arguments. Maybe with an INSTANTIATE_TEST_SUITE_P?

### CODE REVIEW

**Comment by bmagyar** [comment]
Adding more tests would be very welcome! Parametrized tests indeed are the way to go for most of these options. Feel free to scout around the repo and also ros2_controllers for good examples on how to do that!

**Review by bmagyar — APPROVED** [review]

**Comment by Nils-ChristianIseke** [comment]
@bmagyar thanks for the hints.  Seems like i am missing something essential, cause the execution of the spawner fails. Any suggestions what could be wrong?

File: controller_manager/test/test_spawner_unspawner.cpp [hunk 1]
```diff
@@ -274,3 +274,13 @@ TEST_F(TestLoadController, unload_on_kill)
 
   ASSERT_EQ(cm_->get_loaded_controllers().size(), 0ul);
 }
+
+TEST_F(TestLoadController, spawner_test_load_only_in_arg)
+{
+  EXPECT_EQ(call_spawner("ctrl_2 -c test_controller_manager --load-only "), 0);
```
Comment (saikishor) [inline controller_manager/test/test_spawner_unspawner.cpp:280]:
```suggestion
  cm_->set_parameter(rclcpp::Parameter("ctrl_2.type", test_controller::TEST_CONTROLLER_CLASS_NAME));
  EXPECT_EQ(call_spawner("ctrl_2 -c test_controller_manager --load-only "), 0);
```

You can either do this or pass an argument to the spawner

**Comment by Nils-ChristianIseke** [comment]
Closed in favor of #1160

**Pull request closed without merging**